app-conf/GeneralConf.xml

```diff
@@ -45,4 +45,19 @@
     <name>drelephant.analysis.fetch.initial.windowMillis</name>
     <value>3600000</value>
   </property> -->
+  <property>
+    <name>drelephant.analysis.retention.period</name>
+    <value>30</value>
+    <description>Number of days of analysis to be kept in the database</description>
+  </property>
+  <property>
+    <name>drelephant.analysis.purge.batch.size</name>
+    <value>10000</value>
+    <description>Batch size of purge operation</description>
+  </property>
+  <property>
```

Comment:
Do you really need a parameter for `drelephant.analysis.purge.operation.interval`? I would prefer to have a reasonable constant there and introduce a parameter only for a well-defined use-case.